package.json

```diff
@@ -9,17 +9,17 @@
     "test:only": "mate-scripts test",
     "update": "mate-scripts update",
     "postinstall": "npm run update",
-    "test": "npm run lint && npm run test:only"
+    "test": "jest"
```

Comment:
Why did you change it? 
The task doesn't require package.json editing.

Reply:
I just followed the steps shown in the video. Returned the code to its previous state